ops/_main.py

```diff
@@ -49,27 +49,27 @@ def _exe_path(path: Path) -> Optional[Path]:
 
 
 def _create_event_link(
-    charm: '_charm.CharmBase',
+    charm_dir: Path,
```

Comment:
Minor and probably a separate PR: we're supposed to express this as `pathlib.Path`, aren't we?

Reply:
Yes - this was using an existing import, but switching it is fine with me.